aiogram/contrib/fsm_storage/mongo_motor.py

```diff
@@ -104,7 +104,7 @@ async def get_db(self) -> AsyncIOMotorDatabase:
     @staticmethod
     async def apply_index(db):
         for collection in COLLECTIONS:
-            await db[collection].create_index(keys=[('chat', 1), ('user', 1)],
+            await db[collection].create_index(keys=[('chat_id', 1), ('user_id', 1)],
```

Comment:
hmm. such renaming of internal database fields can cause a huge problems to users of MongoDB. Let's keep the internal naming as is